internal/email/awsses/awsses.go

```diff
@@ -12,43 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package email provides the email utilities for minder
-package email
+// Package awsses provides the email utilities for minder
+package awsses
```

Comment:
I figure this is vendor agnostic and others could in time add for example a `gmail` `mailchimp` etc?

Reply:
Yeah. initially I had a wrapper and this interface - 
```go
// Service is the email service interface
type Service interface {
	SendEmail(ctx context.Context, to, subject, bodyHTML, bodyText string) error
}
```
But it felt obsolete since the watermill event message payload now includes everything that is needed for another client to be hooked up. 

If we or someone else wants to add another email service provider, they have to implement the `Register` method (`events.Consumer` interface) and that would be enough.